choreolib/src/main/java/choreo/ext/TriggerExt.java

```diff
@@ -0,0 +1,51 @@
+package choreo.ext;
+
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.VarHandle;
+import java.util.function.BooleanSupplier;
+
+import edu.wpi.first.wpilibj.event.EventLoop;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+
+public class TriggerExt extends Trigger {
```

Comment:
I'd prefer if this extra functionality were contributed to wpilib. Reflection is a concern and I don't really like wrapping the class. Not a blocker to merging, but we should pursue the contribution so we can get rid of this when building against wpilib 2025 betas.

Reply:
yeah TriggerExt should be cut out before 2025 release